spec/00-abstract.md

```diff
@@ -0,0 +1,6 @@
+## Abstract
+
+This specification defines a standard format to propagate context
+information for distributed traces. The specification standardizes how context
+information is sent between services and also defines a means to add
+provider-specific context information. 
```

Comment:
provider -> tracing vendor?

Reply:
not everyone is a vendor that's why I used provider.